os_info/src/info.rs

```diff
@@ -147,6 +150,20 @@ impl Info {
     pub fn bitness(&self) -> Bitness {
         self.bitness
     }
+
+    /// Returns the operating system family. See 'Family' for details.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use os_info::{Info, Family};
+    ///
+    /// let info = Info::unknown();
+    /// assert_eq!(Family::Unknown, info.family());
+    /// ```
+    pub fn family(&self) -> Family {
+        self.family
+    }
```

Comment:
I suppose it makes sense to use match here instead of storing family as a separate field.

Reply:
Yes, I had exactly this in mind. Simply put this match into the `family` method instead of expanding the `Info` struct with an additional field. Do you see some disadvantages in this approach?

Reply:
For BSDs, Windows, and MacOS, that is fine since those categories do not expand often. But for the Linux, this would require everytime a new distro is added as a Type, then this match would have to expand and add another arm. The current method should scale with any number of distros without having to update this function or add a new arm to a match.